aries-site/src/pages/_app.js

```diff
@@ -59,15 +60,159 @@ const backgroundImages = {
+            //if it is older than thirty days ago
+            break;
+          }
+          let temporaryString = data[i].body;
+          if (temporaryString.includes('#### Notifications')) {
+            const indexOfFirstComponent =
+              temporaryString.search('#### Notifications') + 22;
+            const notificationSection = temporaryString.slice(
+              indexOfFirstComponent,
+            );
+            const notificationList = notificationSection.split('\r\n\r\n'); //splits them into an array jumping b/w name, sections, and description
+            const regExp = /\[([^)]+)\]/;
+            for (let j = 0; j < Object.keys(notificationList).length; j += 3) {
+              let temporaryName = notificationList[j].trim();
+              let temporary = regExp.exec(temporaryName);
+              let typeChange = temporary[1].trim();
+              let justName;
+              if (typeChange === 'Update') {
+                justName = temporaryName.slice('8').trim();
+              } else if (typeChange === 'New') {
+                justName = temporaryName.slice('5').trim();
+              }
+              if (justName && !(justName in temporaryHistory)) {
+                let sectionArray = notificationList[j + 1]
+                  .slice(1, -1)
+                  .split('][');
+                let finalSectionlist = [];
+                let action = '';
+                if (Object.keys(sectionArray).length === 1) {
+                  //add an active link if only one section has been updated
+                  action =
+                    '#' +
+                    sectionArray[0].trim().replace(/\s+/g, '-').toLowerCase();
+                }
+                for (let i = 0; i < Object.keys(sectionArray).length; i++) {
+                  finalSectionlist[i] =
+                    sectionArray[i].charAt(0).toUpperCase() +
+                    sectionArray[i].slice(1).toLowerCase();
+                }
+                let newUpdate;
+                tokenName = `${justName
+                  ?.toLowerCase()
+                  .replace(/\s+/g, '-')}-last-visited`;
+                if (window.localStorage.getItem(tokenName)) {
+                  if (
+                    window.localStorage.getItem(tokenName) >
+                    new Date(data[i].merged_at).getTime()
+                  ) {
+                    //history of them visiting the page before, and that visit was after the newest update was sent
+                    newUpdate = false;
+                  } else {
+                    //history of them visiting the page before, but that visit was before the newest update was sent
+                    newUpdate = true;
+                  }
+                } else {
+                  //have never seen the page before
+                  newUpdate = true;
+                }
+                temporaryHistory[justName] = {
+                  type: typeChange,
```

Comment:
`type` already has a native HTML meaning. I'm wondering if for clarity we might be able to modify this to `kind` or `changeKind` etc.